drink/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 //! a running node.
 
 #![warn(missing_docs)]
+#![recursion_limit = "1024"]
```

Comment:
why do we need to increase limit?

Reply:
We don't I think I just copied that from polkadot-sdk as I was initially struggling with the macro to compile.